app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Post details here</h1>
\ No newline at end of file
+<h1>Post details here</h1>
+
+    <div class="posts">
+    <p>Comments: <%= @user.comments.count %></p>
+    <p>Likes: <%= @user.likes.count %></p>
+            <p> <%= @post.title %> </p>
+            <div class ="reactions"> 
+                <span>Comments: <%= @post.comments_counter%> </span>
+                <span>Likes: <%= @post.likes_counter%> </span>
+            </div>
+    </div>
+
+    <div class="comments">
+        <% @post.comments.each_with_index do |comment, index|%>
+            <p>Comment <%=index+1%>: <%= comment.author.name%>,  <%= comment.text%> </p>

```

Comment:
- [ ] Kindly ensure you follow the given wireframe to match that given design

Your design
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/497c7443-b1e0-4074-9b0f-05370e7418f4)

Given template
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/d85cf7f1-da90-48a0-8f48-e4ee5b509baf)

Reply:
- [ ] Also fix your views so that the body of the post shows, right now it does not show

A post has been created
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/2f35fb27-781e-43cd-8495-097c5ddd38a0)

The post is not showing
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/6d93b0d3-180b-4705-8702-2a5f51206fbb)

Reply:
- [ ] Again as requested, please fix your design so as to match the given design, I would advise adding a border to your design as is in the given wireframes so as to help you see how elements are being layed out on the page. Also, fix the post so that it shows, as requested.

Your design
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/41f03863-285a-4637-b7f8-945001ab8f19)

Given wireframe
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/d2a7d08d-1159-4516-9019-bf4b7fae4e9e)
### Note that this is the page that is supposed to show an individual post with all its comments and likes